docs/get-started/python.md

```diff
@@ -0,0 +1,127 @@
+
+### Install the CLI
+To initialize a CDK8s project and auto-generate Kubernetes manifests based on our code, we need the CDK8s CLI:
+
+1. Run the following command to install the CLI using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). For more installation methods, see [Install the CDK8s CLI](cli/installation.md).
+```console
+npm install -g cdk8s-cli
+```
+
+### Create a project
+Next, we’ll initialize a project to create the directory structure, install the CDK8s and CDK8s+ packages and all of the dependencies required to run an application using the [init command](https://cdk8s.io/docs/latest/cli/init/). 
+
+1. In a terminal window, run the `init` command in an empty directory: 
+```console
+cdk8s init python-app
+```
+
+2. In your IDE, open the `main.py` file. 
+```python
+#!/usr/bin/env python
+from constructs import Construct
+from cdk8s import App, Chart
+
+
+class MyChart(Chart):
+   def __init__(self, scope: Construct, id: str):
+       super().__init__(scope, id)
+
+       # define resources here
+
+
+app = App()
+MyChart(app, "python")
+
+app.synth()
+```
+
+This sample shows the basic structure of a CDK8s application with the essential libraries: **constructs** and **cdk8s**. These libraries supply the foundational classes and methods required for working with CDK8s. It includes the following components:
+
+   * A custom `MyChart` class inherits from the [Chart](https://cdk8s.io/docs/latest/reference/cdk8s/python/#chart) base class provided by the cdk8s library, serving as a representation of the Kubernetes resources to be generated and managed.
+   * The `__init__` method within the `MyChart` class is responsible for initializing the base class ([Chart](https://cdk8s.io/docs/latest/reference/cdk8s/python/#chart)) and specifying the Kubernetes resources. This method is invoked when creating an instance of the class.
+   * An instance of the [App](https://cdk8s.io/docs/latest/reference/cdk8s/python/#app) class signifies the primary entry point of the CDK8s application and oversees the application's lifecycle and resources.
+   * An instance of the `MyChart` class is created by passing the app instance and a string identifier, "python", as arguments. This action generates and registers the Kubernetes resources defined in the `MyChart` class within the application. Note that in this example, the "python" string identifier is autogenerated based on the current directory name.
+   * The `synth` method is called on the `app` instance, which produces the required Kubernetes YAML manifest files based on the defined resources. Note that in this example, we haven't defined any resources within the `MyChart` constructor, so running the "cdk8s synth" command in the CLI would generate a blank Kubernetes manifest.
+
+## Defining Kubernetes resources
```

Comment:
For consistency. The two previous sections are framed as a call for action. 

```suggestion
## Define Kubernetes resources
```

Reply:
Updated.